tensorboard/webapp/metrics/views/card_renderer/scalar_card_types.ts

```diff
@@ -77,7 +77,9 @@ export interface PartitionedSeries {
  * value and the DataTable widget must know how to display each value.
  */
 export enum ColumnHeaders {
+  ALIAS = 'ALIAS',
   COLOR = 'COLOR',
+  DISPLAY_NAME = 'DISPLAY_NAME',
```

Comment:
The code does not seem to be working so it's hard to know what you had in mind. . . but I'm confused by the choice to represent ALIAS and DISPLAY_NAME as Columns since you are not displaying them as such.

Should the display name data be tracked in some way that does not intersect/confuse with the concept of "ColumnHeaders" that are being displayed to the user?

Reply:
I tested this on TB.corp so that I could ensure it met your usecase then manually copied the code out to github. "Surprisingly" manually copying lines of code is error prone... apologies for not catching that earlier.

`ColumnHeaders` is simply all the POTENTIAL headers, not the headers being displayed.
These make sense to me as potentially useful in the future though maybe `DISPLAY_NAME` isn't the best name?

If you'd still prefer I avoid using the headers I could move to storing a `RUN -> SortableName` map on the `ScalarCardDataTable`.

Reply:
I suspect the real impetus here is the SelectedStepRunData type. I think ideally we would just add a `displayName` type there (and use `displayName` much like you are currently using `DISPLAY_NAME` for the calculation) but you can't do that since it's a mapped type.

Maybe the best compromise here is to add DISPLAY_NAME but don't add ALIAS. The ALIAS stuff seems likely to be dead code forever.

Reply:
Not adding `ALIAS` makes sense to me.

Reply:
I went ahead with a pretty significant refactor to get away from using headers at all.